src/includes/footer.njk

```diff
@@ -7,11 +7,10 @@
       <a href="/blog/" class="header_link__desktop mb-2">Blog</a>
       <a href="/podcast/" class="header_link__desktop mb-2">Podcast</a>
       <a
-        href="https://docs.orbit.love/docs"
-        target="_blank"
+        href="https://docs.orbit.love/developers"
```

Comment:
shouldn’t that be https://orbit.love/developers?

Reply:
👍🏻